app/src/main/java/com/babylon/wallet/android/presentation/transaction/TransactionReviewViewModel.kt

```diff
@@ -440,6 +453,20 @@ sealed interface PreviewType {
             allTransfers.resources.filter { it.transferable.isNewlyCreated }.map { it.transferable }
         }.flatten()
     }
+
+    data class Stake(
+        val from: List<AccountWithTransferableResources>,
+        val to: List<AccountWithTransferableResources>,
+        val validators: List<ValidatorDetail>
+    ) : PreviewType
+
+    data class Unstake(
+        val from: List<AccountWithTransferableResources>,
+        val to: List<AccountWithTransferableResources>,
+        val validators: List<ValidatorDetail>
+    ) : PreviewType
```

Comment:
Currently seems the same preview type but with a different action like stake/unstake. In my opinion PreviewTypes can become more generic and reveal their differences in their properties. So I would change this Preview Type to `Staking` and keep all these properties which are the same, but add one more for understanding what the action is. I have not seen all the designs but I would argue that claim will look like this too.